src/main/java/controller/board/BoardController.java

```diff
@@ -0,0 +1,75 @@
+package controller.board;
+
+import application.BoardService;
+import dto.MovementDto;
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+import java.util.function.BiConsumer;
+import ui.InputView;
+import ui.output.BoardOutputView;
+
+public class BoardController {
+    private final InputView inputView;
+    private final BoardOutputView outputView;
+    private final BoardService boardService;
+    private final Map<BoardCommand, BiConsumer<List<String>, Integer>> commands;
+
+    public BoardController(InputView inputView, BoardOutputView outputView, BoardService boardService) {
+        this.inputView = inputView;
+        this.outputView = outputView;
+        this.boardService = boardService;
+        this.commands = new EnumMap<>(BoardCommand.class);
+    }
+
+    private void putCommands() {
+        commands.put(BoardCommand.MOVE, this::move);
+        commands.put(BoardCommand.STATUS, (ignore, roomId) -> status(roomId));
+        commands.put(BoardCommand.END, (ignore, roomId) -> {
+        });
```

Comment:
```suggestion
        commands.put(BoardCommand.END, (ignore, roomId) -> {}); 
```
요게 더 한 눈에 들어오는 것 같아요!

Reply:
우테코 컨벤션을 쓰고 있는데 코드 자동 정렬하면 개행이 돼요 🥲

Reply:
헉 이부분은 저도 몰랐네요..!! 😢